datadog_checks_dev/datadog_checks/dev/tooling/commands/env/shell.py

```diff
@@ -0,0 +1,43 @@
+# (C) Datadog, Inc. 2019-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import click
+
+from ...e2e import create_interface, get_configured_envs
+from ...testing import complete_active_checks, complete_configured_envs
+from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info
+
+
+@click.command('shell', context_settings=CONTEXT_SETTINGS, short_help='Run a shell inside agent container')
+@click.argument('check', autocompletion=complete_active_checks)
+@click.argument('env', autocompletion=complete_configured_envs, required=False)
```

Comment:
A nice and convenient thing would be providing `-v/--install-vim`, that will install vim (`apt update; apt install -y vim`) if it's not already there before running the shell.

Reply:
Good idea - can also include `less` as well since I use that extensively

Reply:
Maybe as separate options ? so it can be selective.

Alternative is to have: 
- `ddev env shell --install 'vim less'
- `ddev env shell -i 'vim less'

Reply:
Added `-v` and `-i` options

Reply:
Added examples to PR description